x-pack/plugins/ml/public/ui_actions/apply_influencer_filters_action.tsx

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { ActionContextMapping, createAction } from '../../../../../src/plugins/ui_actions/public';
+import {
+  AnomalySwimlaneEmbeddable,
+  SwimLaneDrilldownContext,
+} from '../embeddables/anomaly_swimlane/anomaly_swimlane_embeddable';
+import { MlCoreSetup } from '../plugin';
+import { SWIMLANE_TYPE, VIEW_BY_JOB_LABEL } from '../application/explorer/explorer_constants';
+import { Filter, FilterStateStore } from '../../../../../src/plugins/data/common';
+
+export const APPLY_INFLUENCER_FILTERS_ACTION = 'applyInfluencerFiltersAction';
+
+export const CONTROLLED_BY_SWIM_LANE_FILTER = 'anomaly-swim-lane';
+
+export function createApplyInfluencerFiltersAction(
```

Comment:
Is it possible to only have this action available if the user only has one row selected in the swim lane? Otherwise with the way filters are `AND`ed on the dashboard, it is likely that other panels will give no results? Unless you can have them added with an `OR`?

![image](https://user-images.githubusercontent.com/7405507/87295852-c2ab5600-c4fd-11ea-81d7-346a17a80f75.png)

Reply:
Made the filter action only available for the single row selection in https://github.com/elastic/kibana/pull/71082/commits/9693776fae7909a55a96a6852acadd9d3684ce4b